src/panel/category/CategoryPanel.jsx

```diff
@@ -32,24 +32,31 @@ export default class CategoryPanel extends React.Component {
   }
 
   componentDidMount() {
-    const { category } = this.props.poiFilters;
-
     this.mapMoveHandler = listen('map_moveend', this.fetchData);
-
-    if (category) {
```

Comment:
As `componentDidUpdate` is not called after the first render, I'm not sure you can safely move that in it.

Reply:
Yes you 're right, and we surely telemetry to be send only on firt open